src/main/java/org/apache/commons/lang3/SystemUtils.java

```diff
@@ -1084,7 +1084,7 @@ public class SystemUtils {
     // -----------------------------------------------------------------------
     // These MUST be declared after those above as they depend on the
     // values being set up
-    // OS names from http://www.vamphq.com/os.html
+    // OS names from http://lopica.sourceforge.net/os.html
```

Comment:
Why? Let's not replace one (insecure) URL with another: "The connection to lopica.sourceforge.net is not secure" Also the comment is no longer true since we did not, in fact, get the data from lopica.

Reply:
I checked on wayback machine and this page was moved to this url. The old domain now hosts the "Headquarter of Vampire Fetishists" ;).

I removed the link since this is not true.